components/database/mariadb-106/Makefile

```diff
@@ -0,0 +1,142 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2011-2013, EveryCity Ltd. All rights reserved.
+# Copyright 2016-2017, Wiselabs Software Ltda - http://www.wiselabs.com.br . All rights reserved.
+# Copyright 2019, Michal Nowak
+# Copyright 2022, erwinlem
+# Copyright 2022, Daniel Bell
+#
+
+BUILD_BITS= 64
+BITS=		64
+BUILD_STYLE=cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		mariadb
+COMPONENT_MAJOR_VERSION=10.6
```

Comment:
10.9.2 is here. Should we ship it instead?

Reply:
I picked 10.6 to package primarily because it's the last long-term release, supported until 2026 (see https://mariadb.org/about/#maintenance-policy). 10.7+ have much shorter lifecycles. Also, I just checked around a bit and CentOS, Fedora, Debian, Ubuntu, Alpine, and FreeBSD are all on 10.5 or 10.6 in their latest distro releases as well.

Reply:
+1 for shipping 10.6.x and to skip 10.7, 10.8, 10.9.

Reply:
OK